client/src/components/NameCard.js

```diff
@@ -21,30 +24,158 @@ const Container = styled.div`
         margin-right:auto;
     }
 
-    #name, #location, #desc{
+    #name, #location, #desc, #email{
         text-align:center;
     }
 
+
+
+
 `;
 
 const RequestButton = styled(Button)`
   width:100%;
 `;
+const useStyles = makeStyles({
+    form:{
+        display:"flex",
+        flexDirection:"column",
+        alignItems:"center",
+        height:"100%",
+        justifyContent:"space-between",
+    },
+    input:{
+        display:'none',
+
+    },
+    cover:{
+        cursor:"pointer",
+        maxWidth:"100%"
+    },
+    profile:{
+        cursor:"pointer",
+    },
 
+    textField:{
+        fontFamily: "Montserrat",
+    }
+
+});
+//TODO: pass in props and get data from props
 export default function Namecard({  }) {
-  return (
+
+    const classes = useStyles();
+
+    const handleChange = name => event => {
+        setValues({ ...values, [name]: event.target.value });
+      };
+    
+    const [values, setValues] = React.useState({
+        name: 'Atsushi Mikaki',
+        location: 'Toronto, Canada',
+        desc:'Chef with 8 years in Japanese cuisine. Reciepient of awards and positive feedback ',
+        
+    }); 
+  
+  
+    return (
 
     <Container>
+        <Switch>
+          {/* change to if else, if user is logged in and  */}
+          {/* this is user's profile page, show edit button */}
+          
+            <Route path="/chef/edit/:chef_id">  
+            <form className={classes.form}>
+                <input
+                    accept="image/*"
+                    className={classes.input}
+                    id="background-img-file"
+                    multiple
+                    type="file"
+                />
+                <label htmlFor="background-img-file">
+                    <img className={classes.cover} alt="background" src="/cover-sushi.png" />
+
+                </label>
+                <input
+                    accept="image/*"
+                    className={classes.input}
+                    id="profile-img-file"
+                    multiple
+                    type="file"
+                />
```

Comment:
We should add something to these two image file inputs to visually represent the fact that they can be clicked and are inputs. Right now they look the same as they do when viewing the profile in read-only mode

Reply:
I will do a hover effect and show a msg like click here to upload new img.